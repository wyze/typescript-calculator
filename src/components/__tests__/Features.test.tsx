import { h } from 'preact'
import Features from '../Features'
import snapshot from './utils/snapshot'

describe('<Features />', () => {
  it('renders', () => snapshot(<Features />))
})
