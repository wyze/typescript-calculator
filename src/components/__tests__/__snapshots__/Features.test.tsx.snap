// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Features /> renders 1`] = `
.glamor-0,
[data-glamor-0] {
  display: -webkit-box;
  display: -moz-box;
  display: -ms-flexbox;
  display: -webkit-flex;
  display: flex;
  flex-wrap: wrap;
  justify-content: space-around;
  -webkit-box-lines: multiple;
  -webkit-flex-wrap: wrap;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-around;
}

<div
  className="glamor-0"
>
  <Feature
    emoji={0}
    text="Simple operations"
  />
  <Feature
    emoji={0}
    text="Decimals"
  />
  <Feature
    emoji={0}
    text="Percentage"
  />
  <Feature
    emoji={0}
    text="Positive/Negative"
  />
  <Feature
    emoji={1}
    text="Advanced options"
  />
  <Feature
    emoji={1}
    text="Operation history"
  />
</div>
`;
